pallets/vector/src/lib.rs

```diff
@@ -241,6 +245,11 @@ pub mod pallet {
 	#[pallet::getter(fn source_chain_id)]
 	pub type SourceChainId<T: Config> = StorageValue<_, u64, ValueQuery>;
 
+	/// Updater that can submit updates
+	#[pallet::storage]
+	#[pallet::getter(fn updater)]
+	pub type Updater<T: Config> = StorageValue<_, H256, ValueQuery>;
```

Comment:
was thinking of adding a preconfigured one since initially is empty, thoughts?

Reply:
Do you mean in the GenesisConfig? not needed, we can set it right after the upgrade

Reply:
yes, well just to be aware of the deployment process.